README.md

```diff
@@ -49,48 +40,44 @@ This list of steps might seem a bit long, but it's really easy to follow – we
         (e.g. `sharedsecret_secret = bb04fb058e2d7fd19c5bdaa129e7883195f73a9c49414a7eXXXXXXXXXXXXXXXX`)
-   4. Restart the Spreed WebRTC server for it to reload its configuration
+   4. Restart the Spreed WebRTC server to reload its configuration
 6. Head over to the **apps/spreedme/config** folder in your Nextcloud installation. Copy `config.php.in` to `config.php` and adjust the constants as already done in `server.conf`:
-   1. Set `SPREED_WEBRTC_ORIGIN` to the origin of your WebRTC server
-      (e.g. `const SPREED_WEBRTC_ORIGIN = 'https://webrtc.mynextcloudserver.com:8443';`)
-      **NOTE:** If you chose option **1.** in the Preparations step above, you can set `SPREED_WEBRTC_ORIGIN` to an empty string: `const SPREED_WEBRTC_ORIGIN = '';`
-   2. Set `SPREED_WEBRTC_BASEPATH` to the same **basePath** you already set in the `server.conf` file
-   3. Set `SPREED_WEBRTC_SHAREDSECRET` to the same **sharedsecret_secret** you already set in the `server.conf` file
+   1. Set `SPREED_WEBRTC_SHAREDSECRET` to the same **sharedsecret_secret** you set in the `server.conf` file before
 7. Head over to the **apps/spreedme/extra/static/config** folder in your Nextcloud installation. Copy `OwnCloudConfig.js.in` to `OwnCloudConfig.js` and adjust the constants as already done in `server.conf` and `config.php`:
-   1. Set `OWNCLOUD_ORIGIN` to the origin of your Nextcloud server
-      (e.g. `OWNCLOUD_ORIGIN: 'https://mynextcloudserver.com:8443'`)
-      **NOTE:** If you chose option **1.** in the Preparations step above, you can set `OWNCLOUD_ORIGIN` to an empty string: `OWNCLOUD_ORIGIN: ''`
 8. Enable this Nextcloud app by browsing to **/index.php/settings/apps**
-9. **That's it.** You can now start communicating securely with your friends and family by opening **/index.php/apps/spreedme** of your Nextcloud host in your browser.
-   For debugging, simply append `?debug` to that URL.
+9. **That's it.** You can now start communicating securely with your friends and family by opening the **Spreed.ME app** of your Nextcloud host in your browser.
 
 ## Limiting access to this app
-- By default, all users who can log in to your Nextcloud installation can also use this app (and spreed-webrtc)
-- If you want to limit access to this app (and spreed-webrtc) only to a selected user-group, open Nextcloud's user configuration site in your browser: **/index.php/settings/users**
-- Create a new group there (e.g. "spreed.me"). For sure you can also use an existing group (like "admin")
+
+- By default, all users who can log in to your Nextcloud installation can also use this app (and Spreed WebRTC).
+- If you want to limit access to this app (and Spreed WebRTC) only to a selected user-group, open Nextcloud's user configuration site in your browser: **/index.php/settings/users**.
+- Create a new group there (e.g. "spreed.me"). For sure you can also use an existing group (like "admin").
```

Comment:
We should recommend creating a `Spreed.ME` group (notice the capitalization) instead.